pkgs/by-name/ma/mathemagix/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchsvn,
+  readline,
+  ncurses,
+  bison,
+  libtool,
+  gmp,
+  mpfr,
+}:
+
+stdenv.mkDerivation {
+  pname = "mathemagix";
+  version = "11126";
+
+  src = fetchsvn {
+    url = "https://subversion.renater.fr/anonscm/svn/mmx/";
+    rev = 11126;
```

```suggestion
    rev = lib.toInt finalAttrs.version;
```

(not necessary, needing finalAttrs)